collect_app/src/main/res/values/buttons.xml

```diff
@@ -1,42 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <style name="Widget.Collect.Button" parent="Widget.Material3.Button">
+        <item name="android:paddingTop">12dp</item>
+        <item name="android:paddingBottom">12dp</item>
+        <item name="android:textAppearance">?textAppearanceBodyLarge</item>
+    </style>
+
+    <style name="Collect.Button" parent="Widget.Material3.Button">
         <item name="android:paddingTop">12dp</item>
         <item name="android:paddingBottom">12dp</item>
         <item name="android:textAppearance">?textAppearanceBodyMedium</item>
     </style>
 
-    <style name="Widget.Collect.Button.Icon" parent="Widget.Material3.Button.Icon">
+    <style name="Collect.Button.Icon" parent="Widget.Material3.Button.Icon">
```

Comment:
I know it ends up being weird because of our own "widget" term, but the idea of having `Widget.` as a prefix here is to differentiate between styles that are uses for views and themes (which use `Theme.`). You'll see the Material and AppCompat styles and themes follow the same pattern. Could we keep `Widget.` and do `Widget.Collect.Button.QuestionWidget` instead?

Reply:
But `Widget` indicates it's used by form elements (questions) and here it's not true because those buttons might be used in other places in the app.

Reply:
Ok, that makes sense, done.